src/super_gradients/training/losses/yolox_loss.py

```diff
@@ -114,11 +150,13 @@ def compute_loss(self, predictions: List[torch.Tensor], targets: torch.Tensor) \
                 obj_target = transformed_outputs.new_zeros((total_num_anchors, 1))
                 fg_mask = transformed_outputs.new_zeros(total_num_anchors).bool()
             else:
+                # GT boxes to image coordinates
                 gt_bboxes_per_image = labels_im[:, 2:6] * self.im_size
                 gt_classes = labels_im[:, 1]
                 bboxes_preds_per_image = bbox_preds[batch_idx]
 
                 try:
```

Comment:
Do we want to keep this? It's supposed to help when during training there is not enough memory for assignment. In this case it will be attempted again, but on CPU. I actually saw it happen a few times

Reply:
Sure, lets keep it.